src/libaktualizr/crypto/crypto.cc

```diff
@@ -341,14 +341,16 @@ StructGuard<EVP_PKEY> Crypto::generateRSAKeyPairEVP(KeyType key_type) {
       return {nullptr, EVP_PKEY_free};
   }
 
-#if AKTUALIZR_OPENSSL_PRE_11
-  StructGuard<RSA> rsa(RSA_generate_key(bits,     /* number of bits for the key - 2048 is a sensible value */
-                                        RSA_F4,   /* exponent - RSA_F4 is defined as 0x10001L */
-                                        nullptr,  /* callback - can be NULL if we aren't displaying progress */
-                                        nullptr), /* callback argument - not needed in this case */
-                       RSA_free);
-#else
```

Comment:
This looks like we're dropping the minimal support for pre-1.1 that we had. Am I misreading it?

Reply:
We're dropping support for pre-0.9.8 - the check against 1.1 was incorrect.

Reply:
Do we have a way to test that 0.9.8 will work? My memory is that xenial is still something like 1.0.2. Although if the xenial tests are passing with this as it is, that's probably good enough. I don't think we've ever tested something before 1.0.

Reply:
If we hadn't tested 0.9.8 before than I don't think it's worth it to start now.